internal/testing/exchange/exchange.go

```diff
@@ -38,11 +37,8 @@ func Setup(e exchange.IBotExchange) error {
 	if err != nil {
 		return fmt.Errorf("LoadConfig() error: %w", err)
 	}
-	parts := strings.Split(fmt.Sprintf("%T", e), ".")
```

Comment:
Just a curiosity, what use-case did this originally address?

Reply:
A way to get the exchange name from the type (e.g btcmarkets.BTCMarkets). The problem though is that for this exchange the name is set to "BTC Markets" via SetDefaults and thus engine also expects "btc markets" https://github.com/thrasher-corp/gocryptotrader/blob/master/engine/helpers.go#L1010

Reply:
Just getting exchange name, without calling `SetDefaults`. Which turned out to not work for several exchanges.